packages/rollup-contracts/contracts/FullStateManager.sol

```diff
@@ -25,7 +46,7 @@ contract FullStateManager is StateManager {
      * @param _slot The slot we're querying.
      * @return The bytes32 value stored at the particular slot.
      */
-    function getStorage(address _ovmContractAddress, bytes32 _slot) internal view returns(bytes32) {
+    function getStorage(address _ovmContractAddress, bytes32 _slot) public view returns(bytes32) {
```

Comment:
I think we should be able to limit some of these functions to being `external` which would be a small performance improvement that might add up.  `external` is cheaper because it uses the calldata directly instead of loading into memory (which it needs to do if it can be called internally)

Reply:
Oh nice catch